guidelines/datanote.md

```diff
@@ -0,0 +1,46 @@
+**_Journal of Cheminformatics_ will only publish research or software that is entirely reproducible by third parties.
+This means that any datasets, software and algorithms that are required to reach the conclusions stated in the
+paper must be provided as supplemental materials, or be otherwise accessible without the need for registration,
+login or agreement with license terms other than Creative Commons licenses for data and text and
+[OSI-approved Open Source Licenses](http://opensource.org/licenses/alphabetical)
+for software. For any software, the source code must be provided.**
+
+A Data Note should describe a curated, CCZero or CC-BY-licensed dataset likely to be of broad utility. The dataset must
+be curated as detailed in this publication and archived for long-term reuse. A Data Note describes the content
```

Comment:
should describe?

Reply:
If you want, I can change to "A Data Note must describe..."

Reply:
Done.